src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiImageBuildInfo.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc. (jcasey@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.koji.model.xmlrpc;
+
+import org.commonjava.rwx.binding.anno.DataKey;
+import org.commonjava.rwx.binding.anno.StructPart;
+
+@StructPart
+public class KojiImageBuildInfo
```

Comment:
Isn't the image build type supported by a koji plugin? It's not part of the core, is it? Why can't we use KojiBuildInfo for this, instead of a unique type?